packages/app/prisma-utils/src/types.ts

```diff
@@ -42,3 +47,16 @@ export type PrismaTransactionReturnType<T> = Either<
   unknown,
   T | runtime.Types.Utils.UnwrapTuple<Prisma.PrismaPromise<unknown>[]>
 >
+
+//----------------------------------------
+// Prisma client builder types
+//----------------------------------------
+/**
+ * this is a temporal solution in the meantime Prisma includes ReadCommitted as a valid isolation level for CockroachDB
+ */
+export type PrismaClientBuilderOptions = Omit<Prisma.PrismaClientOptions, 'transactionOptions'> & {
+  dbDriver?: DbDriver // default: CockroachDb
+  transactionOptions?: Omit<Prisma.PrismaClientOptions['transactionOptions'], 'isolationLevel'> & {
+    isolationLevel?: CockroachDbIsolationLevel
+  }
```

Comment:
Whenever the Prisma client supports `ReadCommitted` we will be able to remove this overwrite, also adding `dBDriver` there in case we want to have different default configs depending on it